app/src/main/java/com/eveningoutpost/dexdrip/ShareModels/DexcomShare.java

```diff
@@ -0,0 +1,133 @@
+package com.eveningoutpost.dexdrip.ShareModels;
+
+import com.eveningoutpost.dexdrip.ShareModels.Models.ExistingFollower;
+import com.eveningoutpost.dexdrip.ShareModels.Models.InvitationPayload;
+import com.eveningoutpost.dexdrip.ShareModels.Models.ShareUploadPayload;
+import com.eveningoutpost.dexdrip.ShareModels.UserAgentInfo.UserAgent;
+import com.squareup.okhttp.ResponseBody;
+
+import java.util.List;
+import java.util.Map;
+
+import retrofit.Call;
+import retrofit.http.Body;
+import retrofit.http.Headers;
+import retrofit.http.POST;
+import retrofit.http.Query;
+
+/**
+ * Created by stephenblack on 3/16/15.
+ */
+public interface DexcomShare {
+
+    @POST("General/LoginPublisherAccountByName")
+    Call<String> getSessionId(@Body Map<String, String> body);
+    //Since this seems to respond with a string we need a delegate that will parse the response body
+    //new String(((TypedByteArray) response.getBody()).getBytes());
+
+    @POST("Publisher/IsRemoteMonitoringSessionActive")
+    Call<Boolean> checkSessionActive(@Query("sessionId") String sessionId);
+    // needs ?sessionId={YourSessionId}
+    // returns true or false
+
+    @POST("Publisher/StartRemoteMonitoringSession")
+    Call<ResponseBody> StartRemoteMonitoringSession(@Query("sessionId") String sessionId,
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method names should not start with capital letters](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1305335957/issues/source?bid=2735053&fileBranchId=2947762#l34)